Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16-alpine
+FROM node:20-alpine
```

Comment:
We need to update the `.nvmrc` file in the project route to reflect `v20` now that we've upgraded Node.